### PR TITLE
fix(ld-select): prop forwarding

### PR DESCRIPTION
**Pull request by borisdiakur**
References:
  - #157

Fixes #(issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

- [x] unit tests
- [x] tested manually

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/liquid/components/ld-select/test/ld-select.spec.ts [hunk 1]
```diff
@@ -2008,4 +2008,26 @@ describe('ld-select', () => {
...
+
+    const ldSelect = page.root
+    const btnTrigger = ldSelect.shadowRoot.querySelector(
+      '.ld-select__btn-trigger'
+    )
+    ;(btnTrigger as HTMLElement).focus = jest.fn(focusManager.focus)
+
+    await page.waitForChanges()
+    jest.advanceTimersByTime(0)
+    expect((btnTrigger as HTMLElement).focus).toHaveBeenCalledTimes(1)
```
Comment (renet) [inline src/liquid/components/ld-select/test/ld-select.spec.ts:2031]: The focusManager does not really do anything here to support the test case. Wouldn't use it then.

```suggestion
    const btnTrigger: HTMLElement = ldSelect.shadowRoot.querySelector(
      '.ld-select__btn-trigger'
    )
    btnTrigger.focus = jest.fn()

    await page.waitForChanges()
    jest.advanceTimersByTime(0)
    expect(btnTrigger.focus).toHaveBeenCalledTimes(1)
```

**Review by renet — CHANGES_REQUESTED** [review]

File: src/liquid/components/ld-select/ld-select.tsx [hunk 2]
```diff
@@ -889,22 +913,22 @@ export class LdSelect implements InnerFocusable {
...
+    const triggerCl = ['ld-select__btn-trigger']
+    if (this.invalid) triggerCl.push('ld-select__btn-trigger--invalid')
+    if (detached) triggerCl.push('ld-select__btn-trigger--detached')
+    if (inline) triggerCl.push('ld-select__btn-trigger--inline')
+    if (ghost) triggerCl.push('ld-select__btn-trigger--ghost')
 
-    let triggerIconCl = 'ld-select__icon'
-    if (this.expanded) triggerIconCl += ' ld-select__icon--rotated'
+    const triggerIconCl = ['ld-select__icon']
+    if (this.expanded) triggerIconCl.push('ld-select__icon--rotated')
```
[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-select/ld-select.tsx:931]: Maybe this is a subjective topic (I think this is readable). Can you please elaborate on how you would improve the code here?

File: src/liquid/components/ld-select/test/ld-select.spec.ts (hunk 1 above)
Comment (borisdiakur) [reply src/liquid/components/ld-select/test/ld-select.spec.ts]: removed

File: src/liquid/components/ld-select/ld-select.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (renet) [reply src/liquid/components/ld-select/ld-select.tsx:931]:
```suggestion
    const cl = [
      'ld-select',
      this.size && `ld-select--${this.size}`,
      this.invalid && 'ld-select--invalid',
      this.expanded && 'ld-select--expanded',
      detached && 'ld-select--detached',
      inline && 'ld-select--inline',
      ghost && 'ld-select--ghost',
    ]

    const triggerCl = [
      'ld-select__btn-trigger',
      this.invalid && 'ld-select__btn-trigger--invalid',
      detached && 'ld-select__btn-trigger--detached',
      inline && 'ld-select__btn-trigger--inline',
      ghost && 'ld-select__btn-trigger--ghost',
    ]

    const triggerIconCl = ['ld-select__icon', this.expanded && 'ld-select__icon--rotated']
```

[the remark this answers is not in the document]
Comment (borisdiakur) [reply src/liquid/components/ld-select/ld-select.tsx]: done

**Review by renet — APPROVED** [review]

**Pull request merged**